source/assets/stylesheets/styles/_contact_page_address.scss

```diff
@@ -0,0 +1,70 @@
+.contact-page-address {
+  line-height: 1.4em;
+}
+
+.contact-page-address__phone-number {
+  padding: 20px 0 15px;
+  font-weight: 300;
+
+  a {
+    color: $dark-gray;
+    text-decoration: none;
+  }
+}
+
+.contact-page-address__email {
+  padding-bottom: 15px;
+  font-weight: 300;
+
+  a {
+    color: $dark-gray;
+    text-decoration: none;
+  }
+}
+
+.contact-page-address__map-link {
+  padding-bottom: 20px;
+  font-weight: 300;
+  border-bottom: solid 1px lighten($medium-gray, 30%);
```

Comment:
This gray is quite dark in comparison the the blog dividing border:

![screen shot 2015-12-18 at 12 17 54](https://cloud.githubusercontent.com/assets/885223/11896794/6287941e-a581-11e5-849e-0077e6531301.png)

![screen shot 2015-12-18 at 12 17 46](https://cloud.githubusercontent.com/assets/885223/11896796/65a1ee24-a581-11e5-8f31-9fd293d5086c.png)